configure.ac

```diff
@@ -1182,6 +1182,23 @@ AS_IF([test "$USE_NLS" = "yes"], [
   AC_MSG_NOTICE([Language support with gettext not available])
 ]) 
 
+#############################################################
+# Sort out fmt
+#############################################################
+
+dnl If in a git repo, get submodule, unless BOUT_DONT_UPDATE_GIT_SUBMODULE is set
+AC_CHECK_FILE([.git], [
+  AS_IF([test "x$BOUT_DONT_UPDATE_GIT_SUBMODULE" == "x"], [
+    git submodule update --init externalpackages/fmt
+  ])
+])
+
+dnl Copy the one file we need to somewhere else
+AC_CONFIG_LINKS(src/fmt/format.cxx:externalpackages/fmt/src/format.cc)
+
+dnl This include must come _first_
+EXTRA_INCS="-I\$(BOUT_TOP)/externalpackages/fmt/include $EXTRA_INCS"
```

Comment:
Similarly, it would be nice to have a way to opt out, instead of patching `configure.ac`